mw-webapp/src/component/button/Button.module.scss

```diff
@@ -1,14 +1,21 @@
 @import "src/styles/variables";
 
 .button {
-  width: 100px;
-  height: 40px;
+  padding: 13px 17px;
+  border: none;
+  border-radius: $primaryBorderRadius;
+  background-color: $inputInitial;
+  color: $whiteText;
+  font-family: $primaryFontFamily;
+  font-size: $primaryFontSize;
+  transition-duration: $primaryTransitionDuration;
 
   &:hover {
+    background-color: $hover;
     cursor: pointer;
   }
 
   &:active {
-    color: rgb(246, 246, 150);
+    background-color: $active;
```

Comment:
let's for color use naming: $primary, secondary, etc

Reply:
`activeColor` is ok too i think

Reply:
Done